packages/chart/src/components/LineChart.tsx

```diff
@@ -36,9 +36,63 @@ export default function LineChart({ xScale, yScale, getXAxisData, getYAxisData,
     return `${formatNumber(value, axis)}`
   }
 
+  const isNumber = value => {
```

Comment:
It looks like this is an example where we're seeing repeated code with isNumber.

Reply:
Yep see my other reply

Reply:
Lifted it up to CdcChart